source/egg/math/Matrix.cc

```diff
@@ -257,7 +261,7 @@ Vector3f Matrix34f::multVector33(const Vector3f &vec) const {
 /// @brief Inverts the 3x3 portion of the 3x4 matrix.
 /// @details Unlike a typical matrix inversion, if the determinant is 0, then this function returns
 /// the identity matrix.
-Matrix34f Matrix34f::inverseTo33() const {
+void Matrix34f::inverseTo33(Matrix34f &out) const {
```

Comment:
Why change the interface?

Reply:
Because otherwise we would be possibly modifying column 3, which we don't want to do. Sure, we're getting rid of default constructors, but returning a stack variable whose 3rd column is UB is possibly different from the base game's behavior of "don't modify the 3rd column"